ipyvolume/widgets.py

```diff
@@ -157,17 +168,23 @@ class Scatter(widgets.Widget):
         ]
     ).tag(sync=True, **texture_serialization)
 
-    material = traitlets.Instance(
-        pythreejs.ShaderMaterial, help='A :any:`pythreejs.ShaderMaterial` that is used for the mesh'
-    ).tag(sync=True, **widgets.widget_serialization)
+    material =  traitlets.Union([
+            traitlets.Instance(pythreejs.ShaderMaterial),
+            traitlets.Instance(pythreejs.MeshPhysicalMaterial),
+            traitlets.Instance(pythreejs.MeshPhongMaterial),
+            traitlets.Instance(pythreejs.MeshLambertMaterial),
+        ], help='A :any:`pythreejs.Material` that is used for the mesh').tag(sync=True, **widgets.widget_serialization)
 
     @traitlets.default('material')
     def _default_material(self):
         return pythreejs.ShaderMaterial()
```

Comment:
```suggestion
        return pythreejs.ShaderMaterial(lights=True)
```
Same here, we could remove the `this.material.lights = true` in scatter.ts